packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+
+import { Modal, Button, Text } from '@rentpath/react-ui-core'
+
+@themed(/^MoreOptionsModal/, { pure: true })
+
+export default class MoreOptionsModal extends PureComponent {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    theme: PropTypes.object,
+    moreOptionsForm: PropTypes.node,
+    total: PropTypes.number,
+    filterTotal: PropTypes.number,
+    onClose: PropTypes.func,
+    onSubmit: PropTypes.func,
+    clearFilters: PropTypes.func,
+    restoreFilters: PropTypes.func,
+  }
+
+  static defaultProps = {
+    theme: {},
+    total: 0,
+    filtersTotal: 0,
+    clearFilters: () => {},
+    onSubmit: () => {},
+    onClose: () => {},
+    restoreFilters: () => {},
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isOpen: props.isOpen,
```

Comment:
Why do we need this? It's not being used anywhere.

Reply:
fixed